Documentation/Books/Manual/Tutorials/Kubernetes/bare-metal.md

```diff
@@ -0,0 +1,532 @@
+<!-- don't edit here, it's from https://@github.com/arangodb/kube-arangodb.git / docs/Manual/ -->
+# ArangoDB on bare metal Kubernetes
+
+A not of warning for lack of a better word upfront: Kubernetes is
+awesome and powerful. As with awesome and powerful things, there is
+infinite ways of setting up a k8s cluster. With great flexibility
+comes great complexity. There are inifinite ways of hitting barriers.
+
+This guide is a walk through for, again in lack of a better word,
+a reasonable and flexible setup to get to an ArangoDB cluster setup on
+a baremetal kubernetes setup.
+
+## BEWARE: Do not use this setup for production!
```

Comment:
Should be:

```
{% hint 'danger' %}
Do not use this setup in production!
{% endhint %}
```

But I would also appreciate a reason as reader, or at least when to use it. I would move to line 18, because it says "test environment" there.